content/2015.12.loadtesting.with.oauth.rst

```diff
@@ -0,0 +1,214 @@
+Loadtesting a HTTP API which uses OAuth for authentication
+##########################################################
+
+:lang: en
+:date: 2015-12-02
+:url: loadtesting-a-http-api-which-uses-oauth-for-authentication
+:summary: How to loadtest an OAuth service relier?
```

Comment:
feedback: a while ago, I would have no idea what a relier means :) Since the summary appears in the homepage, you can be a bit more talkative like «This article explains how to test the performance of an HTTP API when its access depends on an external OAuth service»